lighthouse-core/report/html/templates.html

```diff
@@ -381,20 +381,20 @@
     <a href="" class="lh-topbar__url" target="_blank" rel="noopener"></a>
 
     <div class="lh-tools">
-      <button id="lh-tools-button" class="lh-tools__button" title="Tools menu" aria-label="Toggle report tools menu" aria-haspopup="menu" aria-expanded="false" aria-controls="lh-tools-dropdown">
+      <button id="lh-tools-button" class="lh-tools__button" title="Tools menu" aria-label="Toggle report tools menu" aria-haspopup="menu" aria-expanded="false" aria-controls="lh-tools__dropdown">
```

Comment:
this doesn't [seem right](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html), `lh-tools__dropdown` is just a class, we want the ID to stay, no? Is this related to the icon?

Reply:
oops, global find/replace .... will add back